website/content/docs/getting-started/index.mdx

```diff
@@ -1,7 +1,8 @@
 ---
 layout: docs
 page_title: Get started
-description: Get started with Boundary
+description: >-
+  Learn about the three versions of Boundary to determine which is best for your needs and find more information to help you get started using Boundary.
```

Comment:
"versions" may be confusing, so it's probably safe to spell them all out? The title and URL includes the word, "get started," so you can remove the last part (because of the character limit).

```suggestion
  Understand the difference between HCP Boundary, Boundary Enterprise, and Boundary Community Edition to determine which is best for your needs.
```

Reply:
This is much better, thank you!